### PR TITLE
Properties component in nft full

### PR DESCRIPTION
**Pull request by dblodorn**
Shot to make indexer data a priority and added a handler to convert attributes in object form to an array.

### CODE REVIEW

File: src/nft-full/NFTProperties.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      return (
+        <InfoContainer titleString={'PROPERTIES_TITLE'}>
+          <div {...getStyles("propertiesGrid")}>
+            {formattedAttributes.map((attribute: any, index: number) => {
+              return (
+                <div
+                  {...getStyles("propertiesItem")}
+                  key={`${data?.nft?.tokenId}${index}`}
+                >
+                  {attribute?.trait_type && <span {...getStyles("fullLabel")}>{attribute?.trait_type}</span>}
```
Comment (iainnash) [inline src/nft-full/NFTProperties.tsx:44]: On .co we title-case this – should we titleCase here?

**Comment by iainnash** [comment]
Left a few small comments.

Could you add an example screenshot onto the diff?

File: src/nft-full/NFTProperties.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  const {
+    nft: { data },
+  } = useContext(NFTDataContext);
+  const { getStyles } = useMediaContext();
+  const { metadata } = useNFTMetadata(data?.nft.metadataURI)
+
+  const renderAttributes = (attributes: any) => {
+    
+    function formatAttributes(obj: any) {
+      if (!!obj && obj.constructor === Array) {
```
[the remark this answers is not in the document]
Comment (dblodorn) [reply src/nft-full/NFTProperties.tsx:17]: Yep - this works.

[the remark this answers is not in the document]
Comment (dblodorn) [reply src/nft-full/NFTProperties.tsx:17]: I've added Cryptovoxels Parcel to the Storybook for FullPage - this was an example where the attributes needed to be converted to an array with keys for trait_type & value

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zoralabs/nft-components",
-  "version": "0.1.12",
+  "version": "0.1.13-pre1",
```
Comment (iainnash) [inline package.json:3]: This looks good – although can this line be removed?

**Review by iainnash — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (dblodorn) [reply package.json]: should be up to date now @iainnash

**Review by iainnash — APPROVED** [review]

**Pull request merged**